config/connect-log4j2.properties

```diff
@@ -0,0 +1,39 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name=ConfigConnectConfig
+
+appenders=console, file
+
+appender.console.type=Console
+appender.console.name=STDOUT
+appender.console.layout.type=PatternLayout
+appender.console.layout.pattern=[%d] %p %X{connector.context}%m (%c:%L)%n
```

Comment:
In[ previous config](https://github.com/apache/kafka/blob/trunk/config/connect-log4j.properties#L32-L39), we also set the pattern to ConnectAppender, right?

Reply:
Yes, I think the `ConnectAppender` setting in log4j1 is equivalent to the following, if I haven't misunderstood:

https://github.com/frankvicky/kafka/blob/8ec412ded2272e6c251539b926cfc584a1f0e4ec/config/connect-log4j2.properties#L25-L34

Reply:
Oh, I missed that! Thanks.